client/components/data/query-contact-details-cache/index.jsx

```diff
@@ -24,11 +24,11 @@ class QueryContactDetailsCache extends Component {
 }
 
 QueryContactDetailsCache.propTypes = {
-	requesting: PropTypes.bool,
-	requestContactDetailsCache: PropTypes.func
+	isRequesting: PropTypes.bool.isRequired,
```

Comment:
Go on. As in...  if somehow (and I don't see how) `isRequesting` is unset...  that's ok (so, not ironclad required) because it falls through and makes another request which is what we want in such an unexpected situation?